config/routes.rb

```diff
@@ -45,7 +45,7 @@
   resources :users, only: [:index, :show, :edit]
 
   authenticated :user do
-    root :to => "anthologies#index"
+    root :to => "users#show"
```

Comment:
This branch needs to be rebased, as master contains a new root route, of: `root :to => "anthologies#index"`, among other things.

Reply:
Yes this has been resolved now.